lib/pantry/helpers/category_helper.rb

```diff
@@ -0,0 +1,74 @@
+require 'set'
+
+module Pantry
+
+  module Helpers
+
+    class CategoryHelper
+      class << self
+
+        def categorize(item_name)
+          large_category = determine_large_category(item_name)
+          small_category = helper_for(large_category).categorize(item_name)
+          details = {
+            large_category: large_category,
+            small_category: small_category,
+            time_til_warning: helper_for(large_category).time_til_warning(category: small_category),
+            time_til_expiration: helper_for(large_category).time_til_expiration(category: small_category)
+          }
+        end
+
+        def large_categories
+          categories = [
+              :dairy
+              # :meat,
+              # :poultry,
+              # :fruit,
+              # :vegetable,
+              # :grain,
+              # :packaged,
+              # :other
+          ]
+        end
+
+        private
+
+        def determine_large_category(item_name)
+          categories_max = 0
```

Comment:
Doesn't look like this value ever changes.  If it doesn't, just use `0` wherever you're using this variable.